src/main/java/janggi/moving/Path.java

```diff
@@ -0,0 +1,26 @@
+package janggi.moving;
+
+import janggi.board.position.Position;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Path {
+    private final List<Position> path;
+
+    public Path(List<Position> path) {
+        this.path = new ArrayList<>(path);
+    }
+
+    public List<Position> getIntermediatePath() {
+        int pathSize = path.size();
+        if (pathSize > 2) {
+            return path.subList(1, pathSize - 1);
+        }
+        return new ArrayList<>();
```

Comment:
매직넘버가 보이네요! 앞뒤의 return문 순서를 바꿔서 조건문을 수정해보면 아래처럼 매직넘버를 없앨 수 있어요
```suggestion
        if (path.isEmpty()) {
            return new ArrayList<>();
        }
        return path.subList(1, path.size() - 1);
```

Reply:
기존 로직은
1. 아무것도 없는 경우
2. position이 1개만 들어있는 경우
두 가지 상황에서 빈 리스트를 반환하도록 하려는 의도였는데요!

이프가 제안한 로직은 2번 상황에서 IndexOutOfBounds 예외가 터질 거 같아요.
물론 Path를 생성하기까지의 로직이 제대로 구현되었다면 2번 상황이 발생하지 않을텐데요.
이런 경우에는 믿음으로... 가도 될까요?

우선 매직 넘버를 static final 상수화해주었습니다.😊

Reply:
아니면 Path 생성 시 길이 2 이상이어야 한다는 생성자 검증 로직을 추가하는 것도 방법일 거 같아요.

Reply:
그렇네요! size가 1일 때 예외가 발생하네요ㅠ 말씀주신 듯 생성자 검증 로직 추가가 좋을 듯 해요! Path 객체의 계약 조건으로 해결해도 좋아요~